pkgs/development/haskell-modules/configuration-ghc-8.6.x.nix

```
@@ -95,7 +95,7 @@ self: super: {
   ghc-lib-parser-ex = addBuildDepend self.ghc-lib-parser super.ghc-lib-parser-ex;
 
   # This became a core library in ghc 8.10., so we don’t have an "exception" attribute anymore.
-  exceptions = null;
+  exceptions = super.exceptions_0_10_7;
```

I think this should technically be `self` (which would continue to work in the case where an end-user's overlay modified the `exceptions_0_10_7` package):

```suggestion
  exceptions = self.exceptions_0_10_7;
```

---

Sounds good! `nix-instantiate` agrees that hash did not change.